govc/modules/add_vms.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright (c) 2022 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package modules
+
+import (
+	"context"
+	"flag"
+	"fmt"
+
+	"github.com/vmware/govmomi/govc/cli"
+	"github.com/vmware/govmomi/govc/flags"
+	"github.com/vmware/govmomi/vapi/cluster"
+	"github.com/vmware/govmomi/vim25/mo"
+)
+
+type addVMs struct {
+	*flags.ClientFlag
+	*flags.SearchFlag
+	moduleID string
+}
+
+func init() {
+	cli.Register("modules.add_vms", &addVMs{})
+}
+
+func (cmd *addVMs) Register(ctx context.Context, f *flag.FlagSet) {
+	cmd.ClientFlag, ctx = flags.NewClientFlag(ctx)
+	cmd.ClientFlag.Register(ctx, f)
+
+	cmd.SearchFlag, ctx = flags.NewSearchFlag(ctx, flags.SearchVirtualMachines)
+	cmd.SearchFlag.Register(ctx, f)
+
+	f.StringVar(&cmd.moduleID, "module_id", "", "Module ID")
```

Comment:
Since Module ID is required, this could just be the 1st arg to the command(s). As Usage `ID VM...`
Or if using a flag is preferred, let's just name it `-id`.

Reply:
Renamed to `-id`